editor/editor_properties.cpp

```diff
@@ -3196,6 +3197,13 @@ void EditorPropertyResource::_resource_changed(const Ref<Resource> &p_resource)
 	Ref<ViewportTexture> vpt = p_resource;
 	if (vpt.is_valid()) {
 		r = Object::cast_to<Resource>(get_edited_object());
+		if (r && Object::cast_to<VisualShaderNodeTexture>(r)) {
+			EditorNode::get_singleton()->show_warning(TTR("Can't create a ViewportTexture in a Texture2D node because the texture will not be bound to a scene.\nUse a Texture2DParameter node instead and set the texture in a \"Shader Parameters\" tab."));
```

Comment:
```suggestion
			EditorNode::get_singleton()->show_warning(TTR("Can't create a ViewportTexture in a Texture2D node because the texture will not be bound to a scene.\nUse a Texture2DParameter node instead and set the texture in the \"Shader Parameters\" tab."));
```
Unless there are multiple such tabs?

Reply:
Fair, there's only one such tab, fixed